images/jre/image.yaml

```diff
@@ -3,13 +3,27 @@ versions:
       config: configs/latest.apko.yaml
       extractTagsFrom:
         package: openjdk-17
-        prefix: openjdk-
+        prefix: openjdk-jre
```

Comment:
FWIW the `-jre` suffix was dropped in another PR two weeks ago https://github.com/chainguard-images/images/pull/256/files#diff-3313beffc0b154f1127c0db56645507d074031c287e20c280555f1d9e6e87d36L6.  I think that was driven by a view that it was redundant given `jre` appears in the image name.  @amouat @patflynn is that right?

Reply:
yep that's correct.